<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayerWidget</class>
<widget class="QWidget" name="videoWidget" native="true">
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;</string>
  </property>
  <layout class="QGridLayout" name="videoLayout">
   <item row="0" column="1">
	<spacer name="topMargin">
	 <property name="orientation">
	  <enum>Qt::Vertical</enum>
	 </property>
	 <property name="sizeHint" stdset="0">
	  <size>
	   <width>20</width>
	   <height>40</height>
	  </size>
	 </property>
	</spacer>
   </item>
   <item row="1" column="0">
	<spacer name="leftMargin">
	 <property name="orientation">
	  <enum>Qt::Horizontal</enum>
	 </property>
	 <property name="sizeHint" stdset="0">
	  <size>
	   <width>40</width>
	   <height>20</height>
	  </size>
	 </property>
	</spacer>
   </item>
   <item row="1" column="2">
	<spacer name="rightMargin">
	 <property name="orientation">
	  <enum>Qt::Horizontal</enum>
	 </property>
	 <property name="sizeHint" stdset="0">
	  <size>
	   <width>40</width>
	   <height>20</height>
	  </size>
	 </property>
	</spacer>
   </item>
   <item row="1" column="1">
	<widget class="aske::VideoView" name="videoView" native="true">
	 <property name="palette">
	  <palette>
	   <active>
		<colorrole role="Button">
		 <brush brushstyle="SolidPattern">
		  <color alpha="255">
		   <red>0</red>
		   <green>0</green>
		   <blue>0</blue>
		  </color>
		 </brush>
		</colorrole>
		<colorrole role="Base">
		 <brush brushstyle="SolidPattern">
		  <color alpha="255">
		   <red>0</red>
		   <green>0</green>
		   <blue>0</blue>
		  </color>
		 </brush>
		</colorrole>
		<colorrole role="Window">
		 <brush brushstyle="SolidPattern">
		  <color alpha="255">
		   <red>0</red>
		   <green>0</green>
		   <blue>0</blue>
		  </color>
		 </brush>
		</colorrole>
	   </active>
	   <inactive>
		<colorrole role="Button">
		 <brush brushstyle="SolidPattern">
		  <color alpha="255">
		   <red>0</red>
		   <green>0</green>
		   <blue>0</blue>
		  </color>
		 </brush>
		</colorrole>
		<colorrole role="Base">
		 <brush brushstyle="SolidPattern">
		  <color alpha="255">
		   <red>0</red>
		   <green>0</green>
		   <blue>0</blue>
		  </color>
		 </brush>
		</colorrole>
		<colorrole role="Window">
		 <brush brushstyle="SolidPattern">
		  <color alpha="255">
		   <red>0</red>
		   <green>0</green>
		   <blue>0</blue>
		  </color>
		 </brush>
		</colorrole>
	   </inactive>
	   <disabled>
		<colorrole role="Button">
		 <brush brushstyle="SolidPattern">
		  <color alpha="255">
		   <red>0</red>
		   <green>0</green>
		   <blue>0</blue>
		  </color>
		 </brush>
		</colorrole>
		<colorrole role="Base">
		 <brush brushstyle="SolidPattern">
		  <color alpha="255">
		   <red>0</red>
		   <green>0</green>
		   <blue>0</blue>
		  </color>
		 </brush>
		</colorrole>
		<colorrole role="Window">
		 <brush brushstyle="SolidPattern">
		  <color alpha="255">
		   <red>0</red>
		   <green>0</green>
		   <blue>0</blue>
		  </color>
		 </brush>
		</colorrole>
	   </disabled>
	  </palette>
	 </property>
	 <property name="mouseTracking">
	  <bool>true</bool>
	 </property>
	 <property name="styleSheet">
	  <string notr="true"/>
	 </property>
	</widget>
   </item>
   <item row="2" column="1">
	<spacer name="bottomMargin">
	 <property name="orientation">
	  <enum>Qt::Vertical</enum>
	 </property>
	 <property name="sizeHint" stdset="0">
	  <size>
	   <width>20</width>
	   <height>40</height>
	  </size>
	 </property>
	</spacer>
   </item>
  </layout>
  <widget class="QSlider" name="progressSlider">
   <property name="geometry">
	<rect>
	 <x>0</x>
	 <y>0</y>
	 <width>84</width>
	 <height>19</height>
	</rect>
   </property>
   <property name="focusPolicy">
	<enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
	<string notr="true">QSlider::groove:horizontal {
background: #eee;
height: 2px;
}

QSlider::groove:horizontal:hover {
background: #eee;
height: 4px;
}

QSlider::sub-page:horizontal {
background: white;
height: 2px;
}

QSlider::add-page:horizontal {
background: #777;
height: 2px;
}

QSlider::handle {
width: 0px;
height: 0px;
}</string>
   </property>
   <property name="orientation">
	<enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
	<rect>
	 <x>0</x>
	 <y>0</y>
	 <width>19</width>
	 <height>84</height>
	</rect>
   </property>
   <property name="focusPolicy">
	<enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
	<string notr="true">QSlider::groove:vertical {
background: #eee;
width: 2px;
}

QSlider::groove:vertical:hover {
background: #eee;
width:4px;
}

QSlider::sub-page:vertical {
background: #777;
width: 2px;
}

QSlider::add-page:vertical {
background: white;
width: 2px;
}

QSlider::handle {
width: 0px;
height: 0px;
}</string>
   </property>
   <property name="orientation">
	<enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="codecErrorLabel">
   <property name="geometry">
	<rect>
	 <x>0</x>
	 <y>0</y>
	 <width>241</width>
	 <height>51</height>
	</rect>
   </property>
   <property name="font">
	<font>
	 <pointsize>20</pointsize>
	</font>
   </property>
   <property name="text">
	<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Unknown codec!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
	<set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
  <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>aske::VideoView</class>
   <extends>QWidget</extends>
   <header location="global">videoplayer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>